tests/mgbench/benchmark.py

```diff
@@ -198,87 +242,128 @@ def filter_benchmarks(generators, patterns):
+                print(
+                    "Executing benchmark with",
+                    count,
+                    "queries that should " "yield a single-threaded runtime of",
+                    args.single_threaded_runtime_sec,
+                    "seconds.",
+                )
+                print(
+                    "Queries are executed using",
+                    args.num_workers_for_benchmark,
+                    "concurrent clients.",
+                )
+                memgraph.start_benchmark()
+                ret = client.execute(
+                    queries=get_queries(func, count),
+                    num_workers=args.num_workers_for_benchmark,
+                )[0]
+                usage = memgraph.stop()
+                ret["database"] = usage
 
-            # Output summary.
-            print()
-            print("Executed", ret["count"], "queries in",
-                  ret["duration"], "seconds.")
-            print("Queries have been retried", ret["retries"], "times.")
-            print("Database used {:.3f} seconds of CPU time.".format(
-                  usage["cpu"]))
-            print("Database peaked at {:.3f} MiB of memory.".format(
-                  usage["memory"] / 1024.0 / 1024.0))
-            print("{:<31} {:>20} {:>20} {:>20}".format("Metadata:", "min",
-                                                       "avg", "max"))
-            metadata = ret["metadata"]
-            for key in sorted(metadata.keys()):
-                print("{name:>30}: {minimum:>20.06f} {average:>20.06f} "
-                      "{maximum:>20.06f}".format(name=key, **metadata[key]))
-            log.success("Throughput: {:02f} QPS".format(ret["throughput"]))
+                # Output summary.
+                print()
+                print("Executed", ret["count"], "queries in", ret["duration"], "seconds.")
+                print("Queries have been retried", ret["retries"], "times.")
+                print("Database used {:.3f} seconds of CPU time.".format(usage["cpu"]))
+                print("Database peaked at {:.3f} MiB of memory.".format(usage["memory"] / 1024.0 / 1024.0))
+                print("{:<31} {:>20} {:>20} {:>20}".format("Metadata:", "min", "avg", "max"))
+                metadata = ret["metadata"]
+                for key in sorted(metadata.keys()):
+                    print(
+                        "{name:>30}: {minimum:>20.06f} {average:>20.06f} "
+                        "{maximum:>20.06f}".format(name=key, **metadata[key])
+                    )
+                log.success("Throughput: {:02f} QPS".format(ret["throughput"]))
 
-            # Save results.
-            results_key = [dataset.NAME, dataset.get_variant(), group, test]
-            results.set_value(*results_key, value=ret)
+                # Save results.
+                results_key = [dataset.NAME, dataset.get_variant(), group, test]
+                results.set_value(*results_key, value=ret)
```

Comment:
here we are logging the results

Reply:
I argue we should probably put the test type here, to have both versions for benchmarking when it goes to master